financial-connections/src/main/java/com/stripe/android/financialconnections/ui/components/MultipleEventsCutter.kt

```diff
@@ -0,0 +1,108 @@
+package com.stripe.android.financialconnections.ui.components
+
+import androidx.compose.foundation.Indication
+import androidx.compose.foundation.LocalIndication
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.platform.debugInspectorInfo
+import androidx.compose.ui.semantics.Role
+
+/**
+ * A wrapper around [clickable] that prevents multiple clicks from being registered within a
+ * short time frame.
+ */
+internal interface MultipleEventsCutter {
```

Comment:
Should we call this `MultipleEventsDebouncer`? I haven't heard of the term Cutter before.